app/modules/auth/auth_service.py

```
@@ -133,54 +133,48 @@ async def check_auth(
-                logging.error("DEBUG: Firebase token verification failed: %s", str(err))
+            except (ValueError, FirebaseError):
+                # Handle specific token validation errors
+                # Firebase tokens can raise: InvalidIdTokenError, ExpiredIdTokenError, RevokedIdTokenError
+                # All are subclasses of FirebaseError or ValueError
+                logging.warning("Firebase token verification failed")
                 raise HTTPException(
                     status_code=status.HTTP_401_UNAUTHORIZED,
-                    detail=f"Invalid authentication from Firebase. {err}",
+                    detail="Invalid authentication from Firebase.",
                     headers={"WWW-Authenticate": 'Bearer error="invalid_token"'},
                 )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)